tests/testthat/test_concforced.R

```diff
@@ -257,9 +257,38 @@ test_that("Tgav constraint works", {
     hc <- ssp245()
```

Comment:
should we rename this file to test constraints? instead of test concentration forced?

Reply:
I wondered that too—thanks. Will do.